client-runtime/src/test/java/com/microsoft/rest/v2/RestProxyTests.java

```diff
@@ -528,7 +526,7 @@ public void AsyncPutRequestWithUnexpectedResponseAndExceptionType() {
     private interface Service10 {
         @HEAD("anything")
         @ExpectedResponses({200})
-        HttpBinJSON head();
+        RestResponse<Void, Void> head();
```

Comment:
Why did you decide to change these tests?

Reply:
Can't emit `null` from a stream any more in RxJava 2, so in general HEAD requests that return a body entity type aren't really valid any more.